README.md

```diff
@@ -98,11 +100,15 @@ as to provide HomeKit with Roomba's status only when it requests it.
 Click on any of the items below to expand the corresponding answer.
 
 <details>
-  <summary>I don't know what my Roomba's IP address is</summary>  
```

Comment:
I'm a bit confused about these changes too. These are from a separate PR?

Reply:
I think they only show as changes now since you merged the other PR. I need to merge my branches. I'll fix it.

Reply:
Actually, I'm not sure why this is here...  the current version is correct.